docs/Makefile

```diff
@@ -18,9 +18,3 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-clean:
```

Comment:
I think we'd probably still want a `clean` target in our doc build.

Reply:
I looked into this. As I read it in [GNU.org](https://web.eecs.umich.edu/~sugih/pointers/make.html), "This is an optional rule. It allows you to type 'make clean' at the command line to get rid of your object and executable files." It is not necessary, and I don't want to change what Sphinx created. I can do a `make clean` from the `docs/` folder without this, in order to remove the content of the `_build/` folder.